### ISSUE TITLE
Accessing 'configtimeout' as a setting is deprecated

### ISSUE THREAD

**Issue by seanmil** [voxpupuli/puppet-healthcheck#54]
When running a Puppet run containing `tcp_conn_validator` I saw this warning emitted by Puppet:

```
Warning: Accessing 'configtimeout' as a setting is deprecated.
   (location: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/settings.rb:1165:in `issue_deprecation_warning')
```

and traced it back to this line: https://github.com/voxpupuli/puppet-healthcheck/blob/master/lib/puppet_x/puppet-community/tcp_validator.rb#L31
